std/node/_fs/_fs_mkdtemp.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+import { existsSync } from "./_fs_exists.ts";
+
+export type mkdtempCallback = (
+  err: Error | undefined,
+  directory?: string,
+) => void;
+
+// https://nodejs.org/dist/latest-v15.x/docs/api/fs.html#fs_fs_mkdtemp_prefix_options_callback
+export function mkdtemp(prefix: string, callback: mkdtempCallback): void;
+export function mkdtemp(
+  prefix: string,
+  options: { encoding: string } | string,
+  callback: mkdtempCallback,
+): void;
+export function mkdtemp(
+  prefix: string,
+  optionsOrCallback: { encoding: string } | string | mkdtempCallback,
+  maybeCallback?: mkdtempCallback,
+): void {
+  const callback: mkdtempCallback | undefined =
+    optionsOrCallback instanceof Function ? optionsOrCallback : maybeCallback;
+  if (!callback) throw new Error("No callback function supplied");
```

Comment:
Nit pick, use ERR_INVALID_CALLBACK here like the original implementation, you can find it `std/node/_errors.ts`